### ISSUE TITLE
can not work correctly when convert bmp 32bit  to png 

### ISSUE THREAD

**Issue by kevin-gjm** [lvandeve/lodepng#42]
in file lodepng/examples/example_bmp2png.cpp. Line 93-97 wrong R/G/B/A order

should be :
	  image[newpos + 0] = bmp[bmpos + 2]; 
	  image[newpos + 1] = bmp[bmpos + 1]; 
	  image[newpos + 2] = bmp[bmpos + 0]; 
	  image[newpos + 3] = bmp[bmpos + 3]; 
 when I use this it can work well.

**Comment by kevin-gjm** [comment]
Yes, I test it in other machines.Some work well but some can not.
So I made it all 255 as a temporary solution. But it lost the transparency information.
image[newpos + 3] = 255;

**Comment by DLTech21** [comment]
@kevin-gjm what about convert bmp 1bit to png?
[image_77.bmp.zip](https://github.com/lvandeve/lodepng/files/8194349/image_77.bmp.zip)
